src/themes/dark-theme/color-semantic-tokens.js

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import type {
+  FoundationSemanticColorTokensT,
+  CoreSemanticColorTokensT,
+  CoreExtensionSemanticColorTokensT,
+  SemanticColorTokensT,
+} from '../types.js';
+import {hexToRgb as hexToRgba} from '../../styles/util.js';
+import colors from '../../tokens/colors.js';
+
+const foundation: FoundationSemanticColorTokensT = {
+  primaryA: colors.gray200,
+  primaryB: colors.gray900,
+  accent: colors.blue500,
```

Comment:
if people have these already set as primitives in their current theme will they be overwritten by these keys with the same name?

Reply:
You mean if they create a custom theme and extend it with custom props? Then it depends on how they extend the current theme, should their custom props come last then it overrides what's in the default theme.

Reply:
What would `accent` end up as on the final theme object if I did the following?

```js
createTheme({
  // other primitives...
  accent: 'pink',
});
```

In `createTheme` we combine a bunch of colors:

```js
const theme = {
    colors: {
      ...colorTokensWithOverrides, // user defined primitives
      ...getColorComponentTokens(colorTokensWithOverrides),
      ...getDeprecatedSemanticColors(colorTokensWithOverrides),
      ...semanticColors, // has `accent`, `positive`, etc.
    },
};
```

`semanticColors` has keys for `accent`, `positive`, etc. so wouldn't they override the primitives on the final theme object?

Reply:
I see. The `accent`, etc, will be overriden but not the other props that have value set from the theme primitives. I guess foundation semantic colors can be included in the theme building primitives and the sert of semantic values also built it taking overrides into account. What do you think?

Reply:
yeah, maybe add `primaryA` and `primaryB` to `ColorTokensT`.

then we can use a getter like `getSemanticColors(colorTokensWithOverrides)` which will override the `foundation` values with the passed in primitives.

Reply:
Addressed it in [here](https://github.com/uber/baseweb/pull/2474/commits/4ebed3e526f0c349f7c64e8ebc178d335a6e159c)